binderhub/templates/settings.html

```diff
@@ -0,0 +1,58 @@
+{% extends "page.html" %}
+
+{% block head %}
+{{ super() }}
+<script>
+ {% if submit %} window.submitBuild = true; {% endif %}
+</script>
+<link href="{{static_url("vendor/xterm.js-2.6.0/xterm.css")}}" rel="stylesheet" />
+<script src="{{static_url("vendor/xterm.js-2.6.0/xterm.js")}}"></script>
```

Comment:
Does the settings page want xterm and all the build javascript?

Reply:
Haha, likely no, I just copied `index.html` and modified it. These are leftovers. I also need to clean the javascript.